src/voodoo/conversations/VoodooConversation.ts

```diff
@@ -168,4 +249,116 @@ export default class VoodooConversation {
   ): Promise<messageApi.Envelope> {
     return env
   }
+
+  /**
+   * Creates a new single OneToOneSession for the given contact. Does not publish the invite envelope.
+   *
+   * NOTE: we have to very explicit about what this session is. It's a session between two VoodooInstances,
+   * but the two addresses participating could be the same. So otherAddress is only used as the
+   * payload of the invite, but the invite is encrypted for contact and delivered to the contact.address
+   *
+   * Currently, we generate a random topic for the session, and send an Olm PreKey Message
+   * where the encrypted ciphertext is just the random topic. This is the "invite" message.
```

Comment:
Does this topic serve any purpose other than informing the other session where to listen? Can this topic be dynamic?

Reply:
Nope and yes it can be dynamic in the sense we can constantly spawn more sessions between two Vodozemac accounts. The current code setup actually handles that nicely